src/components/checkout.summarylist.jsx

```diff
@@ -48,7 +48,7 @@ Applied Promotions:&nbsp;
       return (
         <li className="cart-discount">
           <label className="cart-summary-label-col">
-Today's Discount:&nbsp;
+            Today's Discount:&nbsp;
```

Comment:
was the apostrophe okay here? You don't need to substitute with &#39; ?

Reply:
putting this fix in another request, with other lint fixes